crates/bevy_render/src/camera/camera.rs

```diff
@@ -193,6 +193,9 @@ impl Camera {
 
     /// Given a position in world space, use the camera to compute the viewport-space coordinates.
     ///
+    /// Note that the GlobalTransform can be out of sync with the Transform until after
+    /// the TransformSystem::TransformPropagate system runs in the PostUpdate stage.
```

Comment:
Two things:

1. GlobalTransform and other rust symbols should be surrounded in quotes, and a link to the relevant doc should be provided, like that:
```
    /// Note that the [`GlobalTransform`] can be out of sync with the [`Transform`] until after
    /// the `TransformSystem::TransformPropagate` system runs in the `PostUpdate` stage.
```
you can run `cargo doc` locally to see if the change passes without warnings.

2. I think giving an explanation on why this string is relevant for the "`world_to_viewport`" method would be nice. Since I myself encountered the issue, I know exactly why you are adding the paragraph here, but someone else might be dumbfounded. Something like "If it seems that the return value of this method lags behind the actual `world_position` make sure to [an **actionable** reformulation of this paragraph]"

Actionable: that can be followed as if they were instructions

Reply:
I updated the comment. In my particular scenario I ended up transforming the Transform of the camera to a GlobalTransform using `&GlobalTransform::from(*camera_transform)`. But it doesn't seem that should be put in the docs as I can imagine it could have different results based on you situation.